admin-dev/themes/new-theme/webpack.config.js

```diff
@@ -1,110 +1,129 @@
-/**
- * 2007-2017 PrestaShop
- *
- * NOTICE OF LICENSE
- *
- * This source file is subject to the Open Software License (OSL 3.0)
- * that is bundled with this package in the file LICENSE.txt.
- * It is also available through the world-wide-web at this URL:
- * http://opensource.org/licenses/osl-3.0.php
- * If you did not receive a copy of the license and are unable to
- * obtain it through the world-wide-web, please send an email
- * to license@prestashop.com so we can send you a copy immediately.
- *
- * DISCLAIMER
- *
- * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
- * versions in the future. If you wish to customize PrestaShop for your
- * needs please refer to http://www.prestashop.com for more information.
- *
- * @author    PrestaShop SA <contact@prestashop.com>
- * @copyright 2007-2017 PrestaShop SA
- * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
- * International Registered Trademark & Property of PrestaShop SA
- */
-var path = require('path');
-var webpack = require('webpack');
-var ExtractTextPlugin = require('extract-text-webpack-plugin');
+const path = require('path');
+const webpack = require('webpack');
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
 
-module.exports = {
-  entry: [
-    'tether/dist/js/tether.js',
-    'jquery/dist/jquery.js',
-    'jquery-ui/jquery-ui.js',
-    'bootstrap/dist/js/npm.js',
-    'bootstrap-tokenfield/dist/bootstrap-tokenfield.js',
-    'moment/moment.js',
-    'eonasdan-bootstrap-datetimepicker/src/js/bootstrap-datetimepicker.js',
-    'jwerty/jwerty.js',
-    'magnific-popup/dist/jquery.magnific-popup.js',
-    'dropzone/dist/dropzone.js',
-    'typeahead.js/dist/typeahead.jquery.js',
-    'typeahead.js/dist/bloodhound.js',
-    'moment/moment.js',
-    'moment/min/locales.js',
-    'PrestaKit/dist/js/select2.min.js',
-    'PrestaKit/dist/js/bootstrap-switch.min.js',
-    'PrestaKit/dist/js/jquery.pstagger.min.js',
-    'PrestaKit/dist/js/prestashop-ui-kit.js',
-    'PrestaKit/dist/js/jquery.growl.js',
-    'bootstrap-slider/dist/bootstrap-slider.js',
-    'sprintf-js/src/sprintf.js',
-    './js/theme.js'
-  ],
+let config = {
```

Comment:
It seems that you're using an old version of Node. It should be automatically detected, so as seen with @Shudrum no need to precise it here.